pyproject.toml

```diff
@@ -12,23 +12,26 @@ classifiers = [
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
         'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
         'Topic :: Scientific/Engineering :: Artificial Intelligence',
 ]
 keywords = ['deepecho', 'DeepEcho']
 dynamic = ['version']
 license = { text = 'BSL-1.1' }
-requires-python = '>=3.8,<3.12'
+requires-python = '>=3.8,<3.13'
 readme = 'README.md'
 dependencies = [
     "numpy>=1.20.0,<2;python_version<'3.10'",
-    "numpy>=1.23.3,<2;python_version>='3.10'",
+    "numpy>=1.23.3,<2;python_version>='3.10' and python_version<'3.12'",
+    "numpy>=1.26.0,<2;python_version>='3.12'",
```

Comment:
Versions before `numpy>=1.26.0` run into `ModuleNotFoundError: No module named 'distutils'`.

Reply:
Yeah it seems like numpy doesn't support python 3.12 on versions before 1.26 on [PyPi either](https://pypi.org/project/numpy/1.25.2/)